motioneye/mediafiles.py

```diff
@@ -107,18 +106,17 @@
 
 def findfiles(path: str) -> typing.List[tuple]:
     files = []
-    for name in os.listdir(path):
-        # ignore hidden files/dirs and other unwanted files
-        if name.startswith('.') or name == 'lastsnap.jpg':
-            continue
-        pathname = os.path.join(path, name)
-        st = os.lstat(pathname)
-        mode = st.st_mode
-        if S_ISDIR(mode):
-            files.extend(findfiles(pathname))
+    with os.scandir(path) as entries:
+        for entry in entries:
```

Comment:
Why do you create the `entries` variable first, instead of looping through `os.scandir(path)` directly? I would suggest the following:
```py
    for entry in os.scandir(path):
```
This should also make before and after easier to compare, since the code inside the loop had the same indentation.

Same question and suggestion for loop in `_list_media_files()` below.